src/components/dashboard/StatusPanel.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import { withStatusInfo } from '../hoc/status';
+
+import * as websocket from '../../services/websocket';
+import * as statusService from '../../services/status';
+
+import Panel from '../commons/Panel';
+import ServiceList from './ServiceList';
+import Spinner from '../commons/Spinner';
+
+class StatusPanel extends Component {
+  componentDidMount() {
```

Comment:
@mesaugat @pratishshr @bishalshr Should we move all these methods along with the lifecycle ones to the HOC, converting this class to a function? What do you think?